charts/newrelic-prometheus-agent/values.yaml

```diff
@@ -406,5 +406,5 @@ config:
 
   # -- It is possible to include extra scrape configuration in [prometheus format](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).
   # Please note, it should be a valid Prometheus configuration which will not be parsed by the chart.
```

Comment:
@paologallinaharbur  could you add some WARNING notice here explaining that any metadata added on the kubernetes jobs like pod name, or service name will need to be manually again here? 
And perhaps something like , this configuration should be used only if a kubernetes or static job can accomplish the task